### PR TITLE
updated Shibboleth logout; removed non-functioning menu choice

### PR DESCRIPTION
**Pull request by darrowcoucla**
Drupal was not logging out with Shibboleth. Consulted with @akohler and updated the logout link.
Removed a non-functioning menu entry - this may be added back at a later time.

### CODE REVIEW

File: www/sites/all/modules/custom/statsform/statsform.module [hunk 1]
```diff
@@ -380,8 +380,8 @@ function statsform_theme($existing, $type, $theme, $path, $sfTemplate) {
  */
 function logout_request_callback() {
   if( $_SESSION['statsform']['sfShib'] == 1 ){
-    session_destroy();
-    drupal_goto( 'https://shb.ais.ucla.edu/shibboleth-idp/Logout' );
+    // shib logs out drupal (verify)
+    drupal_goto( 'https://www-test.library.ucla.edu/Shibboleth.sso/Logout?return=https://shb.ais.ucla.edu/shibboleth-idp/Logout' );
```
Comment (z3cka) [inline www/sites/all/modules/custom/statsform/statsform.module]: Is there a way to make this environment agnostic? Like maybe:
`drupal_goto( '/Shibboleth.sso/Logout?return=https://shb.ais.ucla.edu/shibboleth-idp/Logout' );`

Comment (akohler) [reply www/sites/all/modules/custom/statsform/statsform.module]: Yes, I recommend using `shib_auth_full_logout_url` instead of hard-coding a url or domain

**Comment by darrowcoucla** [comment]
Updating shib logout

**Comment by z3cka** [comment]
remember, you can just update this PR with a new commit rather than opening a new PR.

**Pull request closed without merging**